docs/src/using-criticalup/creating-sample-manifest.rst

```diff
@@ -0,0 +1,39 @@
+.. SPDX-FileCopyrightText: The Ferrocene Developers
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+
+.. _creating_sample_manifest:
+
+Creating sample manifest
+========================
+
+This chapter describes a way to quickly create a sample ``criticalup.toml`` for your project.
+
+Creating a sample manifest file
+-------------------------------
+
+Running the ``init`` command will create a file ``criticalup.toml`` in the current working directory.
+
+.. code-block::
+
+   criticalup init --release <RELEASE>
+
+
+The ``--release`` flag's value must be one of the product releases listed on the `release
+channels page <https://releases.ferrocene.dev/ferrocene/index.html>`_. For example,
+
+.. code-block::
+
+    criticalup init --release "stable-25.02.0"
+
+.. caution::
+
+    Running this command again will overwrite the existing ``criticalup.toml`` file in the current working directory.
+
+Print the contents only
```

Comment:
Personally, I'd remove this section entirely. The flag is present in `--help` and is self-descriptive, also this flag is very niche.

Reply:
I removed the page.